utils/models.py

```
@@ -0,0 +1,178 @@
+            "text-classification", model=self.model, tokenizer=self.tokenizer
+        )
+        output = pipe(article_body)
+        return {
+            "LABEL_0": Leaning.LEFT,
+            "LABEL_1": Leaning.CENTER,
+            "LABEL_2": Leaning.RIGHT,
+        }[output[0]["label"]]
+
+
+class DistilBertPoliticalBias(Model):
+    def __init__(self):
```

Ruff

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)